tests/AutoReview/FixerTest.php

```diff
@@ -186,6 +187,66 @@ public function testFixersAreFinal(FixerInterface $fixer)
         );
     }
 
+    /**
+     * @dataProvider provideFixerDefinitionsCases
+     */
+    public function testFixersHandleEmptyTokens(FixerInterface $fixer)
+    {
+        $tokens = Tokens::fromCode('');
+
+        try {
+            $fixer->fix(new \SplFileInfo(__FILE__), $tokens);
+        } catch (RequiredFixerConfigurationException $e) {
```

Comment:
if this happen, we do not know that fixer works for empty file

by silencing the error, we lied to our-self that it does